README.md

```diff
@@ -1,9 +1,67 @@
 # Polymer Linter
 
-Catch errors in your Polymer project before even running your code.
+Analyze your Polymer project for errors.
 
-## Work in progress
+## Usage
 
-Polymer Linter is being developed to replace [polylint][1], but is not yet complete.  For a working linter, use [polylint][1].
+The linter can be run in one of two ways: from the command line, or through our editor plugins.
```

Comment:
Per the PR discussion, you may want to specify here that it can be run from within `polymer-cli`, and does not come with it's own CLI interface.

Can't you also run the linter via the JS interface? For someone who's reading the repo README, that might be worth mentioning without removing the nod to editor plugins.

Here's how i'd rewrite, feel free to use or modify:

```
The linter can be run on the command line (with the [Polymer CLI]) or JavaScript. We also support [several editor plugins] for live-linting in your text editor.
```